src/explore-education-statistics-admin/src/components/comments/CommentAddForm.module.scss

```diff
@@ -4,10 +4,14 @@
 .container {
   background: govuk-colour('white');
   border: 1px solid govuk-colour('dark-grey');
-  margin-left: -$dfe-comments-gutter/2;
+  margin-bottom: govuk-spacing(2);
   padding: govuk-spacing(2);
-  position: absolute;
-  top: 0;
-  width: $dfe-comments-width;
-  z-index: 9;
+
+  @media (min-width: 1200px) {
```

Comment:
Using a magic number here. We already have a `$dfe-page-width-wide` variable. Should we be using this? 

If not, we should extract out a variable for this breakpoint.

We've got a similar magic number in other Sass modules in this PR too. Would go through these and refactor them accordingly.

Reply:
I've changed this and the other places to use the standard `govuk-media-query` isntead.